frontend/app/routes/_gcweb-app.personal-information.mailing-address.edit.tsx

```diff
@@ -131,67 +178,101 @@ export default function PersonalInformationMailingAddressEdit() {
-          errorMessage={errorMessages.postalCode}
-        />
-        <InputField
-          id="country"
-          label={t('personal-information:mailing-address.edit.field.country')}
-          name="country"
-          required
-          key={isCopyAddressChecked ? homeAddressInfo?.country : addressInfo?.country}
-          defaultValue={defaultValues.country}
-          errorMessage={errorMessages.country}
-        />
-
-        <div className="flex flex-wrap gap-3">
-          <button id="change-button" className="btn btn-primary btn-lg">
-            {t('personal-information:mailing-address.edit.button.change')}
-          </button>
-          <Link id="cancel-button" to="/personal-information" className="btn btn-default btn-lg">
-            {t('personal-information:mailing-address.edit.button.cancel')}
-          </Link>
-        </div>
-      </Form>
+      {isCopyAddressChecked && (
+        <Form className="max-w-prose" method="post">
+          <p>
+            <strong>{t('personal-information:mailing-address.edit.copy-home-address-note')}</strong>
+          </p>
+          <Address
+            address={homeAddressInfo?.address ?? ''}
+            city={homeAddressInfo?.city ?? ''}
+            provinceState={regionList.find((region) => region.code === homeAddressInfo?.province)?.nameEn}
+            postalZipCode={homeAddressInfo?.postalCode ?? ''}
+            country={countryList.find((country) => country.code === homeAddressInfo?.country)?.nameEn ?? ' '}
+          />
+          <input type="text" id="address" name="address" value={defaultValues.address} readOnly hidden />
+          <input type="text" id="city" name="city" value={defaultValues.city} readOnly hidden />
+          <input type="text" id="province" name="province" value={defaultValues.province} readOnly hidden />
+          <input type="text" id="postalCode" name="postalCode" value={defaultValues.postalCode} readOnly hidden />
+          <input type="text" id="country" name="country" value={defaultValues.country} readOnly hidden />
+
+          <div className="flex flex-wrap gap-3">
+            <button id="change-button" className="btn btn-primary btn-lg">
+              {t('personal-information:mailing-address.edit.button.change')}
+            </button>
+            <Link id="cancel-button" to="/personal-information" className="btn btn-default btn-lg">
+              {t('personal-information:mailing-address.edit.button.cancel')}
+            </Link>
+          </div>
+        </Form>
+      )}
+      {!isCopyAddressChecked && (
+        <Form className="max-w-prose" method="post">
+          <div className="checkbox gc-chckbxrdio">
+            <input id="copy-home-address" type="checkbox" name="ifCopyHomeAddress" checked={isCopyAddressChecked} onChange={checkHandler} />
+            <label id="copy-home-address" htmlFor="copy-home-address">
+              Copy home address
+            </label>
+          </div>
```

Comment:
Adding the "Copy Address" checkbox into the `!isCopyAddressChecked` block would mean that it disappears as soon as it is checked. We should always display the "Copy Address" checkbox in case the user changes their mind.

Reply:
fixed